firestore-bigquery-export/firestore-bigquery-change-tracker/src/bigquery/index.ts

```diff
@@ -82,7 +84,25 @@ export class FirestoreBigQueryEventHistoryTracker
         },
       };
     });
-    await this.insertData(rows);
+    const transformedRows = await this.transformRows(rows);
+    await this.insertData(transformedRows);
```

Comment:
Should we explicitly handle the case where transformedRows is undefined (i.e. transform function exists but throws an error)? What will happen in that scenario now?

Reply:
Silently not writing to the table is probably not a good idea. Silently writing the original row instead of the supposed transformed row is an option (depending on the intention, do we want any row or just no row and have some way of backfilling later?). Final option is to throw an error so instead of logging it in transformRows(), could just let it propogate downstream and have the whole execution fail. I don't know what the standard behavior is for extensions.

Reply:
Hmm, seems like the last option is most in line with what the current behavior is. It makes sense to me to fail there rather than calling insertData() on potentially bad input since insertData()'s behavior could change in the future.